RadixWallet/Features/AppFeature/Overlay/Overlay+View.swift

```diff
@@ -25,21 +25,30 @@ extension OverlayReducer {
 
 private extension StoreOf<OverlayReducer> {
 	var destination: PresentationStoreOf<OverlayReducer.Destination> {
-		func scopeState(state: State) -> PresentationState<OverlayReducer.Destination.State> {
-			state.$destination
-		}
-		return scope(state: scopeState, action: Action.destination)
+		scope(state: \.$destination, action: \.destination)
```

Comment:
This is actually one case where we must use the old version, the compiler goes crazy if we don't